app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -64,6 +65,14 @@ class BrowserActivity : DuckDuckGoActivity() {
             it?.let { webView.loadUrl(it) }
         })
 
+        viewModel.navigation.observe(this, Observer {
+            when (it) {
+                BrowserViewModel.NavigationCommand.LANDING_PAGE -> {
```

Comment:
Optional: It took me a moment to process this perhaps the when/switch syntax is too much for a single option? I think an simple if with a static import is more readable.
```
           if (it == LANDING_PAGE) {
                finishActivityAnimated()
            }
```